spec/ai_spec.rb

```diff
@@ -0,0 +1,10 @@
+require '../ttt/Ai'
+
+describe Ai do
+  context "creates a player" do
+    it "creates a player" do
+      player = Ai.new
+      expect(player).to be_instance_of(Ai)
+    end
+  end
```

Comment:
Another way to write this is:

```rb
context "creates a player" do
  subject { Ai.new }
  it { is_expected.to be_instance_of(Ai) }
end
```

This way you don't have to repeat your test description.